<!--
  1 failure:  :disappointed_rel...
  0 warning: 
  1 messages
  
  DangerID: danger-id-default;
-->

<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  :disappointed_relieved: `npm test` failed. See below for details.
  </td>
    </tr>
  </tbody>
</table>



<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  ```
> titanium-mobile@7.5.0 test /Users/build/jenkins/workspace/dk_titanium_mobile_PR-10456-UL3SBLXIAZBDTC76MRYNFGNP7KBOPQ5EGDEIWRWO76WFQ54SBY6Q
> grunt

Running "appcJs:src:lintOnly" (appcJs) task

Running "eslint:src" (eslint) task

/Users/build/jenkins/workspace/dk_titanium_mobile_PR-10456-UL3SBLXIAZBDTC76MRYNFGNP7KBOPQ5EGDEIWRWO76WFQ54SBY6Q/tests/Resources/ti.media.audioplayer.addontest.js
  24:3   error    Expected space or tab after '//' in comment  spaced-comment
  29:55  warning  'e' is defined but never used                no-unused-vars

✖ 2 problems (1 error, 1 warning)
  1 error, 0 warnings potentially fixable with the `--fix` option.

Warning: Task "eslint:src" failed. Use --force to continue.

Aborted due to warnings.
npm ERR! Test failed.  See above for more details.

```
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

--- a/tests/Resources/ti.media.audioplayer.addontest.js
+++ b/tests/Resources/ti.media.audioplayer.addontest.js
@@ -21,7 +21,7 @@ describe('Titanium.Media.AudioPlayer', function () {
 	});
 
 	it.ios('TIMOB-26533', function (finish) {
-		//Ti.Media.Audio player without url set is crashing while registering for event listener
+		//	Ti.Media.Audio player without url set is crashing while registering for event listener
 		audioPlayer = null;
 		audioPlayer = Ti.Media.createAudioPlayer();
 

